packages/cloud-cognitive/src/components/OptionsTile/OptionsTile.js

```diff
@@ -49,6 +49,7 @@ export let OptionsTile = React.forwardRef(
       locked,
       lockedText,
       onToggle,
+      open = false,
```

Comment:
i think by convention we should keep the default values in `defaultProps`. if you can just remove the value here and `size` and just pop the `open` default value in `defaultProps` that should be good.